keyboards/kbdfans/kbd6x/keymaps/via/rules.mk

```diff
@@ -0,0 +1,3 @@
+VIA_ENABLE = yes
+EXTRAFLAGS += -flto
+TMK_COMMON_DEFS += -DLINK_TIME_OPTIMIZATION_ENABLE
```

Comment:
```suggestion
LTO_ENABLE = yes
```

Reply:
Thanks for the suggestion! This is initially what I had, but fails to compile without https://github.com/qmk/qmk_firmware/pull/8663 pulled into core first. @drashna's suggestion circumvents the compiling issue and still works with VIA according to my test.

Should I leave it as is or revert to this suggestion assuming that other request will be pulled?

Reply:
#8663 has been merged so we should apply the suggestion now.